integration-tests/src/test/java/com/datastax/oss/driver/api/core/metadata/TableOptionsIT.java

```diff
@@ -0,0 +1,92 @@
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.datastax.oss.driver.api.core.CqlIdentifier;
+import com.datastax.oss.driver.api.core.CqlSession;
+import com.datastax.oss.driver.api.core.config.DefaultDriverOption;
+import com.datastax.oss.driver.api.core.cql.ResultSet;
+import com.datastax.oss.driver.api.core.metadata.schema.KeyspaceMetadata;
+import com.datastax.oss.driver.api.core.metadata.schema.TableMetadata;
+import com.datastax.oss.driver.api.testinfra.CassandraRequirement;
+import com.datastax.oss.driver.api.testinfra.ccm.CcmRule;
+import com.datastax.oss.driver.api.testinfra.session.SessionRule;
+import com.datastax.oss.driver.api.testinfra.session.SessionUtils;
+import com.datastax.oss.driver.categories.ParallelizableTests;
+import java.time.Duration;
+import java.util.Map;
+import java.util.Optional;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.rules.RuleChain;
+import org.junit.rules.TestRule;
+
+@Category(ParallelizableTests.class)
+public class TableOptionsIT {
+
+  public static CcmRule ccmRule = CcmRule.getInstance();
+  // disable debouncer to speed up test.
+  private static SessionRule<CqlSession> sessionRule =
+      SessionRule.builder(ccmRule)
+          .withKeyspace(false)
+          .withConfigLoader(
+              SessionUtils.configLoaderBuilder()
+                  .withDuration(DefaultDriverOption.REQUEST_TIMEOUT, Duration.ofSeconds(30))
+                  .withDuration(DefaultDriverOption.METADATA_SCHEMA_WINDOW, Duration.ofSeconds(0))
+                  .build())
+          .build();
+
+  @Rule public TestRule chain = RuleChain.outerRule(ccmRule).around(sessionRule);
+
+  @Test
+  @CassandraRequirement(min = "4.0", description = "required to ensure table options are available")
+  public void ensure_table_options() {
+    CqlIdentifier keyspace = SessionUtils.uniqueKeyspaceId();
+    CqlIdentifier readRepairKey = CqlIdentifier.fromCql("read_repair");
+    CqlIdentifier additionalWritePolicyKey = CqlIdentifier.fromCql("additional_write_policy");
+    CqlSession session = sessionRule.session();
+    String createKeyspace =
+        String.format(
+            "CREATE KEYSPACE %s WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 1}",
+            keyspace);
+    // create keyspace
+    session.execute(createKeyspace);
+    String useKeyspace = String.format("USE %s", keyspace.asCql(false));
+    // connect session to this keyspace.
+    session.execute(useKeyspace);
+
+    Optional<? extends KeyspaceMetadata> originalKsMeta =
+        session.getMetadata().getKeyspace(keyspace);
+
+    // A simple table with read_repair and additional_write_policy options
+    ResultSet rs =
+        session.execute(
+            "CREATE TABLE foo(k int, a text, PRIMARY KEY(k)) WITH read_repair='NONE' AND additional_write_policy='40p'");
+    assertThat(rs.wasApplied()).isTrue();
```

Comment:
`wasApplied()` is for lightweight transactions only, it will always be true for a create statement.